doc/quick-start-guide-and-build.md

```diff
@@ -145,6 +145,32 @@ Before building _LPVS_ from source code, ensure that you have the following prer
   sudo apt install mysql-server
   ```
 
+  
+Note: for installing SCANOSS on Ubuntu 2023.04, Fedora 38, Debian 11, etc. or later versions of mentioned OSes-a few additional steps are required, as previously mentioned command : "pip3 install scanoss" will not work correctly. 
```

Comment:
suggestion to move this note **before** step with installing through "pip3 install scanoss".
User will follow instruction step-by-step.

Reply:
May be better to make a reference to note as these instructions are long and will interfere with normal way of instructions?
For example:  For installing SCANOSS  on Ubuntu 2023.04, Fedora 38, Debian 11, etc. or later versions of mentioned OSes please follow note at the end of the paragraph.

Reply:
sounds good, I agree.